### PR TITLE
GETDEL command support

### PR DESCRIPTION
**Pull request by clarity0**
#18

### CODE REVIEW

**Comment by clarity0** [comment]
For some reason that i can't figure out, a redis client request to the server after calling send_packaged_command hangs.
Redis client documentation says it changes the state of the connection?

**Comment by evoxmusic** [comment]
> For some reason that i can't figure out, a redis client request to the server after calling send_packaged_command hangs.
> Redis client documentation says it changes the state of the connection?

Did you pull the main with the latest version?

File: redisless/src/server.rs [hunk 1]
```diff
@@ -421,6 +432,31 @@ mod tests {
...
+
+        // Changing the body of the match for GET to the same one for GETDEL 
+        // and then calling con.get, emulating a getdel, makes it work as you would expect
+        // But for some reason, it doesnt work with con.send_packed_command
+        // The value only gets deleted after having called with another con.get
+
+        let _ = con.send_packed_command(cmd("GETDEL").arg("key").get_packed_command().as_slice());
+        let _: Option<String> = con.get("key").ok(); // works after calling con.get once more
+        let x: Option<String> = con.get("key").ok();
+        assert_eq!(x, None);
```
Comment (clarity0) [inline redisless/src/server.rs:456]: Can't figure out why this is the case

**Comment by evoxmusic** [comment]
It's probably good practice to check if a command is implemented into a popular driver like Python before supporting it. I see hundreds of Redis commands. Implementing all of them is probably a waste of time. What do you think @clarity0 ?

**Comment by barkanido** [comment]
@clarity0 @evoxmusic AFAIK the rust client does support it in the form of raw commands:
```rust
fn do_something(con: &mut redis::Connection) -> redis::RedisResult<String> {
    redis::cmd("GETDEL").arg("my_key").query(con)
}

**Pull request closed without merging**